common/app/views/fragments/containers/facia_cards/commercialContainer.scala.html

```diff
@@ -1,149 +1,58 @@
-                            </a>
-                        </div>
-                        <div class="commercial__body">
-                            @for(layout <- container.containerLayout) {
-                                @for(slice <- layout.slices) {
-                                    <ul class="lineitems l-row l-row--cols-@math.min(Commercial.container.numberOfItems(container), 4)">
-                                    @for(ColumnAndCards(_, cards) <- slice.columns) {
-                                        @for(FaciaCardAndIndex(_, card, hideUpTo) <- cards) {
-                                            @card match {
-                                                case contentCard: ContentCard => {
-                                                    <li class="lineitem l-row__item l-row__item--span-1">
-                                                        <div class="rich-link tone-paidfor--item">
-                                                            <div class="rich-link__container">
-                                                                @for(displayElement <- contentCard.displayElement) {
-                                                                    @displayElement match {
-                                                                        case InlineVideo(videoElement, _, _, _) => { @itemImage(videoElement.images) }
-                                                                        case InlineImage(images) => { @itemImage(images) }
-                                                                        case _ => { }
-                                                                    }
-                                                                }
-                                                            <div class="rich-link__header">
-                                                            @title(contentCard.header, 0, container.index)
-                                                            </div>
-                                                            <div class="rich-link__standfirst u-cf">@for(text <- contentCard.trailText) {@Html(text)}</div>
-                                                            <a class="rich-link__link u-faux-block-link__overlay" @Html(contentCard.header.url.hrefWithRel)>@RemoveOuterParaHtml(contentCard.header.headline)</a>
-                                                            </div>
-                                                        </div>
-                                                    </li>
-
-                                                }
-                                                case _ => {}
-                                            }
-                                        }
-                                    }
-                                    </ul>
-                                }
+<div class="fc-container fc-container--commercial">
```

Comment:
The rest of this template is all going to go as well in the next few weeks so doesn't seem worth changing it.  It's just that we can't show multiple logos on containers until the other work is done.